commands/host.vegman

```diff
@@ -224,6 +224,86 @@ function cmd_virtualmedia_umount {
   usb-ctrl eject "usb.ms.cli"
 }
 
+
+function loc_config_default_set {
```

Comment:
No need to call them `loc_*`. Anything that doesn't start with `cmd_` is safe. These scripts are anyway sourced into `clicmd`.